src/pages/contact.js

```diff
@@ -83,25 +83,12 @@ class ContactUs extends Component {
 
   render() {
     const { name, email, message, submitting, success } = this.state
-    const site = this.props.data.site
     const page = this.props.data.allContentfulPage.edges[0].node
     const { location } = this.props
```

Comment:
might as well gather the data.site on this line, as well

Reply:
Made it all easier by just getting one and not all